### ISSUE TITLE
add ion torrent tutorial

### ISSUE THREAD

**Issue by antgonza** [biocore/qiita#1517]
A user just added and processed fine an Ion Torrent data in Qiita. The user followed these instructions:

For processing in QIIME, suggest taking a look and following the
pipeline described by George Watts in
https://groups.google.com/forum/#!topic/qiime-forum/NEotQnTQlQo. Note
that this pipeline has a few of things: (1) it starts with the per
sample files, (3) it doesn't have QC of the reads, and (2) it doesn't
remove primers. The primers can be left there but in general we always
remove them. 

For Qiita, you could upload the demultiplexed fastq files and then
process them following this tutorial:
http://qiita.ucsd.edu/static/doc/html/tutorials/no-raw-sequences.html#per-sample-fastq-files-without-barcode-and-primer-information
-- Basically, you will need to upload each of the files, add a column
in your prep template name run_prefix, which should match the name of
the file, and use the "per sample fastq" option.

**Comment by antgonza** [comment]
This is already possible: process via per_sample_FASTQs
